pkg/collector/corechecks/snmp/config.go

```diff
@@ -225,6 +233,19 @@ func buildConfig(rawInstance integration.Data, rawInitConfig integration.Data) (
 		c.oidBatchSize = defaultOidBatchSize
 	}
 
+	var bulkMaxRepetitions int
```

Comment:
🤏  wat about initialise it to `int(defaultBulkMaxRepetitions)` and remove the last `else` ?

Reply:
I have a preference of having default value as "fallback" at the end, just a matter of preference :)